config/config.py

```diff
@@ -40,6 +40,13 @@ def build_config():
         ],
     }
 
+    config["output"] = {
+        "mt": [
+            p.q.pt_1,
```

Comment:
this looks strange, either:
```
producer.q.pt_1
```
or 
```
p.quantity.pt_1
```
or even 
```
producer.quantity.pt_1
```

Reply:
Maybe import p.q separately as q?

Reply:
done. Please check again.